website/docs/components/cards.mdx

```diff
@@ -47,3 +47,80 @@ Let op de volgende dingen als u cards gebruikt:
 
 - NN/g Nielsen Norman Group. Cards: UI-Component Definition Geraadpleegd op 20 maart 2020, van https://www.nngroup.com/articles/cards-component/
 - U.S. Web Design System (USWDS). Cards contain content and actions about a single subject. Geraadpleegd op 20 maart 2020, van https://designsystem.digital.gov/components/card/
+
+## Card
+
+### Default
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="default" />
+
+### With Button
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="with-button" />
+
+### With Label
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="with-label" />
+
+### With Selectable And Button
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="with-selectable-and-button" />
+
+### With Slide Toggle
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="with-slide-toggle" />
+
+### With Toggletip
+
+<StorybookComponent name="card" implementations={["core", "html-css", "react"]} variant="with-toggletip" />
+
+Een Card mag uitsluitend in een <Link to="./card-container#card-list">Card List</Link> of <Link to="./card-container#card-grid">Card Grid</Link>
```

Comment:
Waarom is hier `<Link>` voor nodig, kunnen we niet met een normale Markdown link een verwijzing maken? Zoals in `button.mdx` ?